work/worker.go

```diff
@@ -702,27 +702,27 @@ CommitTransactionLoop:
 		switch err {
 		case blockchain.ErrGasLimitReached:
 			// Pop the current out-of-gas transaction without shifting in the next from the account
-			logger.Trace("Gas limit exceeded for current block", "sender", from)
+			logger.Trace("Gas limit exceeded for current block", "sender", from.String())
```

Comment:
Dosen't it call `String` method without calling it explicitly?

Reply:
Oops.^^
I though `common.address` is same with `common.hash` which prints the terminal string (`000000…000000`).
However, as you said, `common.address` can prints full string.
I will revert this. Thanks.